### DISCUSSION TITLE
2023-07-21

### DISCUSSION THREAD

**Discussion by yeopto** [woowacourse-teams/2023-haru-study#121] [daily_meeting]
### 오늘의 할 일
- 배포 마무리 언제까지 할지 정하기
- 3차 스프린트 기획을 얼마나 잡을지(피드백 바탕)

### 언젠가 해야할 일
- 기획 변경된 이유 정리
- DNS 어떻게 할지
- UX 저자 워크숍 사전 미션(~7/25 화) -> 준이 제공한 내용 참고해 데모 끝나고 월요일에 하기로 결정

### 언젠가 해야할 일
- 기획 변경된 이유 정리
- DNS 어떻게 할지
- UX 저자 워크숍 사전 미션(~7/25 화) -> 준이 제공한 내용 참고해 데모 끝나고 월요일에 하기로 결정

**Comment by nlom0218** [comment]
### 체크인

9점 - 배가 계속 아파서 -1

### 2차 스프린트 KPT 회고

K: 기획부분에서는 1차 스프린트 때보다 빠른 의사결정과 실행으로 기능 개발에 들어갈 수 있어서 좋았다. 개발부분에서는 서로 어려워하고 의견을 가지는 부분에 대해 자유롭게 의견을 나누었던 부분이 좋았다.
P: 시간에 대한 규칙이 명확히 있지 않다.
T: 개발에 집중하고 싶은 시간이 있다. 이런 시간을 정하는 것이 어떨까? PR를 보내면 정해진 시간 내에 리뷰를 하는 규칙을 정하는 것도 있으면 좋을 것 같다.

### 우리팀의 협업 점수 + 점수를 더 올리기 위해 시도해볼 사항들

9점

1. 하루 진행상황을 간단하게 공유할 수 있는 시간을 정하기
2. 기술적으로 잘 안되는 부분을 공유할 수 있는 시간을 정하기
3. 코드 리뷰에 대한 마감 시간 정하기

### 체크아웃

10점 - 회식때 맛난거 먹는다.

**Comment by woo-jk** [comment]
### 체크인
10점 - 데모데이 발표가 끝나서 기분이 너무 좋다~ 딱히 큰 지적도 없고 좋은 피드백 많이 받아서 좋다 ㅎㅎ

### 2차 스프린트 KPT 회고
K: 개발 할 때 분업이 효율적으로 잘 되고있는 것 같다.

P: 
1. 프론트-백 끼리 서로 진행 상황이 어떻게 되가는지 잘 모른다. 이슈 칸반 보드를 봐야 현재 다른 사람의 진행상황을 알 수 있는데, 이 칸반 보드에 이슈가 한 두개가 아니다보니 쉽게 파악하기 힘들고 내 이슈가 아닌 이상 잘 안보게 된다. 
2. wiki에 올라오지 않는 문서들은 모두 노션 캘린더에서만 올라오고 있어서 보기 불편하다.

T:  
1. 하루 데일리 미팅같은 시간에 현재 진행 상황을 서로 공유해주면 좋을 것 같다.
2. 노션에 캘린더만 활용하지 말고 다른 페이지도 적극적으로 사용하면 좋을 것 같다.

### 협업 점수
9점 - 찐한 협업 만큼은 우리가 제대로 경험하고 있는 것 같다. 진행 상황 공유가 잘되고, 문서화도 잘되면 최고의 팀이 될 듯


### 체크아웃
10점 - 체크인 때 느낀 감정 그대로라 10점 ㅎㅎ

**Comment by aak2075** [comment]
### 체크인

10점 - 어제 잘 자서 컨디션 굳 2차 스프린트 끝나서 빨리 놀고싶어요

### 2차 스프린트까지의 중간 KPT 회고

K
기획을 변경하느라 개발 시간이 3~4일 정도로 매우 적었고, 이로 인해 완성된 서비스를 배포하지는 못했다. 그래도 데모데이 발표에서 우리의 서비스의 가치에 대해 공감을 많이 얻은것을 확인하고 기획 변경에 시간을 쏟기를 잘했다는 생각이 든다. 시간이 걸리더라도 방향이 잘못 되었다 생각하면 다시 생각해보는 것이 좋았다.

P
1. 문서라는 산출물이 조금 부족한 것 같다. 우리의 기획 의도, 기술적인 문서, 트러블 슈팅을 하는 과정등 문서가 부족하다.
2. 밥먹는 시간이 불규칙적이다.

T
1. 문서를 작성하는 시간을 따로 갖는것도 필요하지만 문서를 작성하는 시간을 어느정도 일정에 포함시키는건 어떨까요?
2. 밥먹을 시간쯤 되면 흐름이 조금 끊기더라도 먹고 와서 하는게 어떨까요

### 우리팀의 협업 점수 + 점수를 더 올리기 위해 시도해볼 사항들

협업 점수 : 9점

정말 많은 이야기를 나누고 페어프로그래밍도 하면서 찐한 협업을 했다는 것을 느낄 수 있었고 서비스가 만들어지는 과정이 즐거웠다. 다만, 일정이 빠듯했어서 그런지 문서화가 부족하다는 느낌이 든다. 회의를 통해 무엇을 문서화할지 정해서 이슈로 할당해야할 것 같다.

### 체크아웃

10점 - 2차 스프린트가 성공적으로 마무리 되어서 좋다. 회식 갑시다~

**Comment by woosung1223** [comment]
### 체크인
9점 - 발표 그래도 성공적으로 잘 한 것 같아서 기분이 좋슴다

### 2차 스프린트까지의 중간 KPT 회고
K: 기능이 모두 변경되었음에도 빠르게 기획과 개발을 진행한 점이 매우 좋았던 것 같아요. 사실 1, 2주차에서 논의했던 내용들을 전부 갈아끼우는거라 두려움이 있었던 것은 사실인데 팀원들이 짧은 기간동안 빠르고 정확하게 논의를 이끌어줘서 나온 결과물이라고 생각합니다. 

P: 어느정도의 개인 시간 보장도 필요할 것 같아요. 데모데이가 가까워지거나 하는 날에는 저녁도 먹지 않고 개발에만 몰두하는 경우도 있었는데, 이런 부분은 좋지 않은 문화로 자리잡을 수 있을 것 같아서 주의가 필요하다고 생각합니다. 

T: 저녁 먹는 시간을 정해놓으면 어떨까요? 데일리 미팅 때 `오늘은 몇 시까지 하고 밥을 먹자.` 처럼 일정을 미리 잡아두어도 괜찮을 것 같아요.

### 우리팀의 협업 점수 + 점수를 더 올리기 위해 시도해볼 사항들

협업 점수: 10점

이유: 우테코에 들어오기 전 세 차례정도 협업을 경험해봤는데, 이정도로 찐한 협업은 처음입니다. 저희 팀원들 모두 `우린 원양어선이야`라고 이야기 할 정도로 협업에 몰두했기 때문입니다. 4주 동안 평일 개인 시간이 거의 없었을 정도로 찐한 협업을 경험했던 것 같아요. 하지만 아무래도 찐한 협업을 경험하다보면 감정이 상하는 순간 등이 찾아오기 마련인데, 그런 순간들이 닥쳐도 나름대로 잘 해결했던 것 같습니다. 다만 1. 해결한 문제에 대한 기록 및 문서화하는 역량을 기르고 2. 개인 자율성을 최소한이라도 보장할 수 있는 팀이 된다면 더 좋은 협업을 할 수 있을 것 같아요.

### 체크아웃 
9점 - 발표도 나름 성공적으로 끝나고, 금요일이라 기분이 괜찮습니다!

**Comment by yeopto** [comment]
### 체크인

10점 - 금요일, 데모데이 끝, 나쁘지 않았던 피드백 3합 좋다.

### 2차 스프린트 까지 KPT 회고
- K - 함께 4주간 맞춰온 팀의 바이브를 계속 유지하면 좋을 것 같다. 4주동안 7명 모두의 각자 성향을 파악하며 팀이 좋은 방향으로 나아갈 수 있게 서로가 서로에게 맞춰졌다고 생각한다. 이대로 유지한다면 좋은 결과를 낼 수 있다는 자신감이 생긴다.
- P - 일정 추정에 대해서는 조금 아쉬운 것 같다. 하는데 까지 최선을 다하긴 하지만 시간관리를 잘 하지 못한 점이 아쉽다.
- T - 일정 추정을 명확하게 한다면 시간을 좀 더 효율적으로 쓸 수 있지 않을까 싶다. 나 같은 경우는 좀 우선순위가 좀 더 높은 일이 있는데 낮은 일에 시간 투자를 많이 하다보니 일이 밀렸다. 일정 추정을 명확히 한다면 하나의 약속이라는 생각에 좀 더 포기할건 포기할 수 있지 않을까 싶다. 

### 우리팀의 협업 점수 + 점수를 더 올리기 위해 시도해볼 사항들
10점 - 협업은 정말 잘 되는 것 같다. 각자 맡은 일에 최선을 다하고, 적절하게 역할 분배도 하고, 코드리뷰를 통해 서로가 프로덕션 코드에 대해 동기화를 하고 있다. 또, 협업하면서 트러블도 없다. 대-만-족 하고 우리의 `원양어선`은 계속 항해한다.

### 체크아웃

10점 - 회식 할 생각에 기분 좋다.

**Comment by MoonJeWoong** [comment]
### 체크인
10점 - 생각했던대로 피드백들을 받아서 기분이 좋네요. 다음 스프린트도 일정 잘 추산해서 성공적으로 마무리할 수 있었으면 좋겠습니다!

### 2차 스프린트까지의 중간 KPT 회고

K:
- 2차 스프린트까지 목표로 설정했던 사항들을 일정에 맞게 유연하게 변경하면서 잘 지켜내온 것.
- 기획을 중간에 변경했는데 오히려 더 서비스를 성공적으로 뾰족하게 만들 수 있었던 것.
- 기획적인 내용이든, 기술적인 내용이든 팀원 모두와 논의하고 결론을 도출하는 것.

P:
- 팀에서 진행하는 업무 중 일정을 맞추기 위해 어쩔 수 없이 분업을 하게 되는 내용들이 있다. 팀원 간 다른 인지 상태를 동기화하는 과정을 거쳐야 협업이 될 수 있을 텐데 이런 과정을 가질 수 있는 시간적 여유가 없는 것 같다.
- 우리 팀 문화로 개발이든 비개발이든 모든 작업 내용을 잘 문서화하기로 했는데 일정 속도 맞추는 것 때문에 실질적으로 문서화가 거의 안되고 있는 것 같아 많이 아쉽다.
- 일정에 쫓기다보니 데일리 미팅이나 회고를 흐지부지 가져갔던 점이 아쉽다.

T:
- 일과 시간 내에 오늘 진행한 사안들에 대해서 문서화하고 정리해서 공유할 수 있는 시간을 따로 두면 좋을 것 같다.
- 아무리 일정이 바쁘더라도 데일리 미팅, 회고 시작 시간은 고정해두고 반드시 지키도록 노력한다. 일정 상 많이 급하다면 소요 시간을 단축하더라도 고정된 시간에 모여서 미팅 및 회고를 진행하면 좋을 것 같다.

### 우리팀의 협업 점수 + 점수를 더 올리기 위해 시도해볼 사항들
9점 - 팀원 간의 소통, 현재 프로젝트의 기획과 나아가고 있는 방향 모두 맘에 든다. 하지만 우리가 바쁜 일정에 항상 쫓기다보니 함께 지키고자 했었던 사항들을 놓치게 되는 양상이 너무 아쉬웠던 것 같다. 원래 지키기로 했던 팀 문화가 현재 일정에 너무 버겁다면 조금 축소해서라도 지킬 수 있는 방향으로 나아갔으면 좋겠다.(문서화, 분업했던 사항들에 대한 동기화 등...) 다른 크루들도 얘기를 해줬지만 개인 일정을 소화할 수 있는 시간을 팀적으로 확보해주는 것이 필요하다는 생각도 들었다.

### 체크아웃
10점 - 2주차 스프린트 기간에 힘들었던 만큼 팀원들 다같이 만족할 수 있는 결과를 얻어낸 것 같다. 이번 회고에서 나온 아쉬웠던 점들까지 잘 보완해서 더 나은 하루스터디 팀이 되었으면 좋겠다!
